runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -328,6 +325,56 @@ public void onChatMessage(ChatMessage event)
 		}
 	}
 
+	@Subscribe
+	public void onGameObjectSpawned(GameObjectSpawned event)
+	{
+		final GameObject gameObject = event.getGameObject();
+		final WorldPoint bombLocation = gameObject.getWorldLocation();
+
+		switch (gameObject.getId())
+		{
+			case ObjectID.CRYSTAL_BOMB:
+				bombs.put(bombLocation, new CrystalBomb(gameObject));
```

Comment:
Additionally you need to remove the bombs on despawn event, and also whenever the game state changes to LOGGED_IN.